webapp/channels/src/components/at_mention/at_mention.test.tsx

```diff
@@ -227,52 +226,4 @@ describe('components/AtMention', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
-
-    test('should have placement state based on ref position of click handler', () => {
```

Comment:
Why the removal of the placement tests? We should be testing for this to make sure it still opens correctly

Reply:
The test doesn't work with functional components with React hooks. I tried to see if I could update it for hooks, but was spending way too much time on it without any result. It was really just testing whether the component's `placement` state was updated to the correct string. That string comes from a utility function, which wasn't tested. So instead of banging my head on testing for updated state, created tests for making sure the utility function outputs the correct string.